pkg/proxmox/goproxmox/api_client.go

```diff
@@ -142,10 +142,18 @@ func (c *APIClient) FindVMResource(ctx context.Context, vmID uint64) (*proxmox.C
 	return nil, fmt.Errorf("unable to find VM with ID %d on any of the nodes", vmID)
 }
 
-// FindVMTemplateByTags tries to find a VMID by its tags across the whole cluster.
-func (c *APIClient) FindVMTemplateByTags(ctx context.Context, templateTags []string) (string, int32, error) {
-	vmTemplates := make([]*proxmox.ClusterResource, 0)
+func isNotInAllowedNodes(allowedNodes []string, node string) bool {
+	return !slices.Contains(allowedNodes, node)
+}
```

Comment:
I'm not sure if this helper is all that useful, `slices.Contains` is pretty clear in its meaning and `isNot`-kind of routines tend to confuse people.

Reply:
good point.